File: codigos/1 - estrutura sequencial/1020 - Idade em Dias/Main.c
// Online C compiler to run C program online
#include <stdio.h>

int main() {
    int dias;
    scanf("%d", &dias);
    
    int tempos_val[] = {365, 30, 1};
    char* tempos[] = {"ano(s)", "mes(es)", "dia(s)"};
    
    for (int i = 0; i < 3; i++) {
        int tempo_val = tempos_val[i];
        char* tempo = tempos[i];
        int qntd = dias / tempo_val;
        printf("%d %s\n", qntd, tempo);
        dias -= qntd * tempo_val;
    }
}

File: codigos/1 - estrutura sequencial/1020 - Idade em Dias/Main.py
from math import floor

dias = int(input())
saidas: list[int] = []
for val, tempo in zip(tempos_val:=[365, 30, 1], tempos:=["ano(s)", "mes(es)", "dia(s)"]):
    qntd_tempo = floor(dias/val) if dias >= val else 0
    print(f"{qntd_tempo} {tempo}")
    dias -= qntd_tempo * val